js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoDocument.js

```diff
@@ -32,25 +32,23 @@ window.ArangoDocument = Backbone.Collection.extend({
     });
   },
 
-  addDocument: function (collectionID, key) {
+  addDocument: function (collectionID, key, body) {
     var self = this;
-    self.createTypeDocument(collectionID, key);
+    self.createTypeDocument(collectionID, key, body);
   },
 
-  createTypeEdge: function (collectionID, from, to, key, callback) {
+  createTypeEdge: function (collectionID, from, to, key, body, callback) {
```

Comment:
Fine, but maybe add a check that the body is a valid object. Otherwise 
```
body._key = key;
```
and so on will fail.